float64 rayon
float64 demi_ecart
float64 u
float64 w

